app/models/firmware_registry_simple.rb

```diff
@@ -0,0 +1,59 @@
+class FirmwareRegistrySimple < FirmwareRegistry
+  def sync_fw_binaries_raw
+    remote_binaries.each do |binary_hash|
+      binary = FirmwareBinary.find_or_create_by(:firmware_registry => self, :external_ref => binary_hash['id'])
```

Comment:
This feels like something we could use graph refresh for treating the registry as the manager

Reply:
We can do that in a follow up though I won't hold this up for that

Reply:
I've asked @Ladas for some guides how to employ graph refresh on non-ems classes, but it should really be done as a followup IMAO. Because quite some files will have to be added (at least collector, parser, persister, definitions etc.).